app/components/CredentialCard.tsx

```diff
@@ -54,12 +54,7 @@ export default function CredentialCard({
             type="button"
             className="usa-button--secondary"
           >
-            <span
-              className="display-inline-block position-relative margin-right-05 height-0"
-              style={{ width: '1em' }}
-            >
-              <Icon.Delete className="left-0 bottom-0 position-absolute" />
-            </span>
+            <Icon.Delete className="bottom-aligned margin-right-05" />
```

Comment:
Looks like this reverts #206. Bad merge?

Reply:
Definitely possibly, is the span correct? I can revert that change

Reply:
I'm not sure. Please show me a screen shot of the client credential segmented cards with this patch.

Reply:
Not sure how that happened, fixing now

Reply:
![image](https://user-images.githubusercontent.com/11023698/185452181-7daa9a6c-9b3a-42cb-b707-24fe1966c8a9.png)

Reply:
Can the Add button be right-aligned with the Delete buttons?

Reply:
Tried to get it close in the latest commit, only for the tablet size though, since it moves around on smaller screens
![image](https://user-images.githubusercontent.com/11023698/185463109-5acacdb7-d816-4fd5-8efd-994b212905b9.png)